sdk/core/azure-core/test/nlohmann-json-test/CMakeLists.txt

```diff
@@ -45,10 +45,17 @@ endif()
 #############################################################################
 
 set(files
+    src/make_test_data_available.hpp
+    src/test_utils.hpp
```

Comment:
It seems like the only uses of `test_utils.hpp` are in test files that we are not including (we commented those out).

https://github.com/search?q=repo%3Anlohmann%2Fjson+test_utils.hpp&type=code

unit-bson.cpp
unit-ubjson.cpp
unit-bjdata.cpp
unit-cbor.cpp
unit-msgpack.cpp

Can we remove this file rather than checking it in?

Also, this header files doesn't seem to be included in the nlohmann.json CMakeLists.txt directly. Do you know how nlohmann is including these headers?

Reply:
good catch, done